actions/dump-charm-artifacts/README.md

```diff
@@ -0,0 +1,24 @@
+## Summary
+
+Dumps logs relevant to kubernetes-based Juju charms to a Github Artifact.
+
+Logs are both printed to screen and dumped to the artifact "juju-kubernetes-charmcraft-logs".
+
+Prerequisites for running this action in a workflow:
+* kubectl (with a kube config for the relevant cluster)
+* Juju bootstrapped on the runner
+
+Warning: This dumps a lot of details from your kubernetes cluster and juju models.
+Before sending this information to anyone, you should inspect it to ensure no
+sensitive information is being shared.
+
+## Example usage
+
+TODO: Complete the path below once this is placed in a longterm location.
+
+```yaml
+- uses: canonical/kubeflow-ci/dump-charm-artifacts@version
```

Comment:
artifacts sounds like *.charm files or whatever. From the name will be hard to get that we are talking about logs, statuses, etc

can we rename?

Reply:
Yeh, I think I agree on this one, `dump-charm-status-artifacts` or `dump-charm-debug-artifacts`?

Reply:
renamed to `dump-charm-debug-artifacts` - still has artifacts in it though, that work for you @beliaev-maksim ?

Reply:
yes, sounds good